.github/workflows/test.yml

```diff
@@ -34,7 +34,14 @@ jobs:
         working-directory: geo-types
     strategy:
       matrix:
-        container_image: ["georust/geo-ci:rust-1.51", "georust/geo-ci:rust-1.52", "georust/geo-ci:rust-1.53"]
+        container_image:
+          # We aim to support at least the last 6 months of rust toolchains, so
+          # run on MSRV and the most recent two releases.
+          #
+          # We omit some of the middle releases for faster CI runs.
+          - "georust/geo-ci:rust-1.51"
```

Comment:
If we had an MSRV of 1.52, it'd still be within a 6 month threshold, but it seemed like we might as well maintain support for 1.51 if tests are already passing.

Also, I realize this "6 month support" policy could be controversial. In practice we've been doing something pretty close to this anyway, but I'd be open to revisiting it if it proves arduous.

Reply:
Very happy to throw it open for wider discussion, but the 6 month policy seems reasonable to me (as someone who admittedly doesn't depend on my distro's Rust version)

Reply:
I would slightly prefer something more precise than "6 months", like "stable - 3".

Reply:
> I would slightly prefer something more precise than "6 months", like "stable - 3".

Good idea - that would be easier to think about. Updated!